recipe/meta.yaml

```
@@ -690,22 +695,27 @@ outputs:
         - {{ pin_subpackage("libarrow-gandiva", exact=True) }}
         - {{ pin_subpackage("libarrow-substrait", exact=True) }}
         - {{ pin_subpackage("libparquet", exact=True) }}
+        - clangdev {{ llvm_version }}
+        - llvmdev {{ llvm_version }}
+        - cython
+        - numpy
+        - python
+        - setuptools
+        # see https://github.com/apache/arrow/issues/37931
+        - setuptools_scm <8
```

That issue should have been solved for v16, it would be good to remove the cap (also for other outputs)
```suggestion
        - setuptools_scm
```